Scheduler/feature_scheduler/auxtel/feb_2025_fbs_spec_survey.py

```diff
@@ -0,0 +1,205 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from astropy import units
+from astropy.coordinates import Angle
+from lsst.ts.fbs.utils import Target
+from lsst.ts.fbs.utils.auxtel.surveys import (
+    generate_cwfs_survey,
+    generate_image_survey_from_target,
+    generate_spectroscopic_survey,
+    get_auxtel_targets,
+)
+from rubin_scheduler.scheduler.detailers import DitherDetailer
+from rubin_scheduler.scheduler.schedulers import CoreScheduler
+
+
+def get_scheduler():
+    """Construct feature based scheduler.
+
+    Returns
+    -------
+    nside : int
+        Healpix map resolution.
+    scheduler : Core_scheduler
+        Feature based scheduler.
+    """
+    # Information for all surveys
+    nside = 64
+    avoid_wind = False
+    if avoid_wind:
+        wind_speed_maximum = 13.0  # maximum direct wind in m/s
+    else:
+        wind_speed_maximum = 30
+
+    # Detailers - for now, just simple image or spectroscopy detailers.
+    image_detailers = [
+        # Dither up to half the FOV, per exposure.
+        DitherDetailer(max_dither=(7 / 2 / 60), per_night=False)
+    ]
+    spec_detailers = []
+
+    # Get target information - edit YAML file for updates
+    # Yaml file is in ts_fbs_utils :
+    # ts_fbs_utils/python/lsst/ts/fbs/utils/data/auxtel_targets.yaml
+    target_pointings = get_auxtel_targets()
+
+    # The targets below must be in the target_pointings
+    # But can be specified to be all or just a subset
+
+    # Imaging priority - high priority imaging, tier 1
+    imaging_priority_targets = ["Photo08000-1"]
+    # Imaging backup - low priority imaging, backup for everything - tier 3
+    imaging_backup_targets = ["Photo08000-1"]
+
+    # Spectroscopy priority - high priority spectroscopy - tier 1
+    spectroscopy_priority_targets = [
+        "WD_0621-376",
+        "HD60753",
+        "HD42525",
+        "HD38666",
+        "WD_0859-039",
+    ]
+    # Backup spectroscopy - tier 2
+    spectroscopy_backup_targets = ["HD185975"]
+
+    # CWFS - tier 0
+    cwfs_time_gap = 120.0  # Gap between cwfs images, in minutes
+    cwfs_block = "BLOCK-305"
+
+    cwfs_survey = generate_cwfs_survey(
+        nside=nside,
+        time_gap_min=cwfs_time_gap,
+        wind_speed_maximum=wind_speed_maximum,
+        cwfs_block_name=cwfs_block,
+    )
+
+    # Go through imaging targets ('auxtel_imaging_targets' category)
+    imaging_priority = []
+    imaging_backup = []
```

Comment:
Same comment here, given that we have spectroscopy backup we can do away with the imaging_backup survey all together.